rustls-platform-verifier/src/verification/mod.rs

```diff
@@ -74,14 +74,17 @@ fn invalid_certificate(reason: impl Into<String>) -> rustls::Error {
     )))
 }
 
-#[cfg(any(windows, target_os = "android"))]
 /// List of EKUs that one or more of that *must* be in the end-entity certificate.
 ///
 /// Legacy server-gated crypto OIDs are assumed to no longer be in use.
 ///
 /// Currently supported:
 /// - id-kp-serverAuth
 // TODO: Chromium also allows for `OID_ANY_EKU` on Android.
+#[cfg(target_os = "windows")]
+const ALLOWED_EKUS: &[*mut u8] = &["1.3.6.1.5.5.7.3.1\0".as_ptr() as *mut u8];
```

Comment:
💭 raised #128 containing an idea on making this clearer.

Reply:
💭 also when our msrv allows it, we could use a `c""` string here

Reply:
Yeah c string literals would be really nice here 😅